### PR TITLE
Improve SAML mapper configuration descriptions

### PR DESCRIPTION
**Pull request by jvlcek**
https://bugzilla.redhat.com/show_bug.cgi?id=1438545

This PR expands upon the descriptions regarding mapper configurations
for SAML. It also removes a section that is no longer needed.

### CODE REVIEW

**Comment by jvlcek** [comment]
@abellotti and @gtanzillo please review

**Comment by jvlcek** [comment]
@miq-bot add_label bug

**Comment by abellotti** [comment]
LGTM!!  Thanks @jvlcek for enhancing our SAML docs.

**Comment by jvlcek** [comment]
Thank you @abellotti 

 @adahms when you get a chance, can you review/merge. Thanks!!

File: auth/saml.adoc [hunk 1]
```diff
@@ -182,12 +180,12 @@ to be SAML aware and function accordingly.
...
 
 * Set mode to External (httpd)
-* Check: _Enable SAML_ - This will enable the SAML login button on the login screen, the redirects
-to the SAML protected page for authentication as well as supporting the SAML logout process.
 * Check: _Enable Single Signon_ - With this option enabled, initial access to the Appliance Administrative UI will
 redirect to the SAML Identity Provider authentication screen.  Note that logouts from the Appliance will
 return the user to the Appliance login screen allowing them to login as admin unless _Disable Local Login_
 is checked below.
+* Check: _Enable SAML_ - This will enable the SAML login button on the login screen, the redirects
+to the SAML protected page for authentication as well as supporting the SAML logout process.
```
Comment (adahms) [inline auth/saml.adoc]: Recommend changing 'authentication as well as supporting' to 'authentication, and supports' to improve clarity.

File: auth/saml.adoc [hunk 2]
```diff
@@ -182,12 +180,12 @@ to be SAML aware and function accordingly.
 Login as admin, then in _Configure->Configuration->Authentication_
 
 * Set mode to External (httpd)
-* Check: _Enable SAML_ - This will enable the SAML login button on the login screen, the redirects
-to the SAML protected page for authentication as well as supporting the SAML logout process.
 * Check: _Enable Single Signon_ - With this option enabled, initial access to the Appliance Administrative UI will
 redirect to the SAML Identity Provider authentication screen.  Note that logouts from the Appliance will
 return the user to the Appliance login screen allowing them to login as admin unless _Disable Local Login_
 is checked below.
+* Check: _Enable SAML_ - This will enable the SAML login button on the login screen, the redirects
```
Comment (adahms) [inline auth/saml.adoc]: s/This will enable/This enables/

Comment (jvlcek) [reply auth/saml.adoc]: Thank you @adahms. Good suggestions. Will do

File: auth/saml.adoc (hunk 1 above)
Comment (jvlcek) [reply auth/saml.adoc]: Thank you @adahms again. Another good suggestions. Will do

**Comment by jvlcek** [comment]
@adahms I've addressed your comments and squashed my commits down to one. Please let me know if you spot other issues you would like me to address. Thank you for the help!

JoeV

**Comment by jvlcek** [comment]
@adahms Bump. Can this merged?

**Comment by adahms** [comment]
@jvlcek - Looks good to me; merging!

**Pull request merged**